src/Microsoft.Extensions.ApiDescription.Design/Microsoft.Extensions.ApiDescription.Design.csproj

```diff
@@ -42,11 +42,6 @@
       <PackagePath>tools/dotnet-getdocument.dll</PackagePath>
       <StrongName>$(AssemblySigningStrongName)</StrongName>
     </SignedPackageFile>
-    <SignedPackageFile Include="../dotnet-getdocument/bin/$(Configuration)/netcoreapp2.1/publish/dotnet-getdocument.exe">
```

Comment:
Since this is a platform-neutral executable, there's a few options:
1. Change the `.targets` file to use the executable and otherwise leave everything as-is
2. Change the `.nuspec` file to exclude this `.exe` file from the `.nupkg`
3. Disable creating the executable file, as you've done here

I'm :ok: with the approach you've chosen but would prefer we go with (1). Let's chat&hellip;

Reply:
1 does not work. The binary has to be run as a standalone. Running `dotnet dotnet-get-document.exe` results in a failure:

```
$ dotnet dotnet-getdocument.exe
Error:
  An assembly specified in the application dependencies manifest (dotnet-getdocu                                                                                                                                                                                                                                             ment.deps.json) has already been found but with a different file extension:
    package: 'dotnet-getdocument', version: '3.0.0-alpha1-t000'
    path: 'dotnet-getdocument.dll'
    previously found assembly: '/home/prkrishn/publish/dotnet-getdocument.exe'
```

Leaving this as is

Reply:
That wasn't my suggestion. Instead change the `.targets` file to run `/home/prkrishn/publish/dotnet-getdocument.exe`. It's currently finding the `.dll` and running it (assuming `dotnet` is in the `$env:Path`).

Reply:
i.e. don't use `dotnet` anymore

Reply:
That doesn't work on xplat. It's not a executable on other platforms.

Reply:
Ah well. Thought that was the point of dotnet/cli#6237.